package-lock.json

```diff
@@ -16347,7 +16347,7 @@
     },
     "minimist": {
       "version": "1.2.5",
-      "resolved": "https://npm.morph.int.tools.bbc.co.uk/minimist/-/minimist-1.2.5.tgz",
```

Comment:
🤣

Reply:
anytime I do an npm update on install I see this :/